exercises/Anne-Hyacinthe/short-response.md

```diff
@@ -0,0 +1,12 @@
+# Unit 2 - Lesson 3: Introduction to JavaScript Functions
+## Short Response Questions
+
+1. What is the difference between function parameters and arguments?
+    A parameter is a placeholder for an argument while an argument is the variable being assessed.
+2. What happens when you invoke a JavaScript function with a missing argument? What about calling a function with _extra_ arguments? 
+    When invoking a function with no argument it will use a _space_ as that value. While a function with an _extra_ argument will get an error. 
```

Comment:
This is completely wrong. Read [this](https://stackoverflow.com/questions/47083910/why-passing-extra-arguments-to-a-function-works-in-javascript) and try this question again.